stable/prometheus-to-sd/values.yaml

```diff
@@ -0,0 +1,16 @@
+replicaCount: 1
+image:
+  repository: gcr.io/google-containers/prometheus-to-sd
+  tag: v0.2.1
+  pullPolicy: IfNotPresent
+resources:
```

Comment:
We usually recommend not to specify defaults for resources and leave this a conscious choice of the user.